recipes/milksnake/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "milksnake" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: dfcd43b78bcf93897a75eea1dadf71c848319f19451cff4f3f3a628a5abe1688
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - cffi
+  run:
+    - cffi
+    - python
+
+test:
+  imports:
+    - milksnake
+    - milksnake.ffi
+
+about:
+  home: http://github.com/simplejson/simplejson
+  license: Apache-2.0
+  license_family: Apache
```

Add the LICENSE to the recipe directory and reference it here
```suggestion
  license_family: Apache
  license_file: LICENSE
```